Issue template (#48)
SuperDARN/pydarn: .github/ISSUE_TEMPLATE/bug_template.md

# BUG 

*description of the bug*

## Priority

- [ ] Minor - typo, incorrect naming, fails on specific uncommon situation

> - [ ] Major - miss representation of data, failure in more common situation 
> - [ ] Urgent - failure in usage, large miss representation

Reviewer: ```suggestion
- [ ] Major - misrepresentation of data, failure in a common situation 
- [ ] Urgent - failure in usage, large misrepresentation
```

## Example of the bug 

*please provide **code** and **output** of the bug*

### Attempts

*What have you tried already to fix it?*

*What have you tried to get around it?*

### Data Location

> *Please indicate where to get the data you found the bug with.*

Reviewer: ```suggestion
*Please indicate where (and how) to get the data file that was used to obtain the bug.*
```

## Potential Bug Location 

*Please note this is not required.*

> *If you can link to some code in pyDARN where you think the bug is coming from this will help the developer fix it sooner* 

Reviewer: ```suggestion
*If you can link to some code in pyDARN where you think the bug is coming from this will help the developer fix it sooner.*
```

## Potential Solution(s)

*Please note this is not required.*

> *If you have some idea of how to fix/improve on the bug then please describe here for possible solution(s)*

Reviewer: ```suggestion
*If you have some idea of how to fix/improve on the bug then please describe here for possible solution(s).*
```

## Extra Notes

> *Please provide any other details about this feature*
> - *Operating System*
> - *python version 3.X.X*
> - *examples of what the visual should look like*
> - *other data files that show the same bug*
> - *How to access data files to test out the bug*

Reviewer: ```suggestion
*Please provide other pertinent details about this feature:*
- *Operating System*
- *Python version 3.X.X*
- *Examples of what the visual output should look like*
- *Other data files that show the same bug*
- *How to access data files to test out the bug*
```
